exercise-analyst-server/API/API.csproj

```diff
@@ -9,6 +9,7 @@
       <PackageReference Include="AutoMapper.Extensions.Microsoft.DependencyInjection" Version="8.1.1" />
       <PackageReference Include="Microsoft.AspNetCore.Authentication.JwtBearer" Version="3.1.14" />
       <PackageReference Include="Microsoft.AspNetCore.Identity.EntityFrameworkCore" Version="3.1.14" />
+      <PackageReference Include="Microsoft.AspNetCore.Mvc.Versioning" Version="5.0.0" />
```

Comment:
W sumie to jak jest .net core w 3.1 to ta paczka tez może być 3.1 - ale jak uważasz

Reply:
Nie mam pojęcia czy wersje paczki to wersje frameworka

Reply:
Dowiemy się pewnie przy deployu bo hosting oferuje wsparcie dla 3.1 ale dla dotneta 5 juz nie :V